tests/data/placebo/test_rds_has_pending_maintenance/rds.DescribeDBInstances_1.json

```diff
@@ -0,0 +1,11686 @@
+{
+    "status_code": 200,
+    "data": {
+        "DBInstances": [
+            {
+                "DBInstanceIdentifier": "pitpp008",
```

Comment:
there's alot of extraneous data in this file (11k lines). please trim to what's needed for the test.